debugtools/DDR_VM/src/com/ibm/j9ddr/view/dtfj/image/J9DDRImageProcess.java

```diff
@@ -83,6 +83,7 @@ public class J9DDRImageProcess implements ImageProcess {
 	private final static int J9PORT_SIG_FLAG_SIGFPE_DIV_BY_ZERO 	= 0x40020;
 	private final static int J9PORT_SIG_FLAG_SIGFPE_INT_DIV_BY_ZERO = 0x80020;
 	private final static int J9PORT_SIG_FLAG_SIGFPE_INT_OVERFLOW 	= 0x100020;
+	private final static int J9PORT_SIG_FLAG_SIGUSR2 	= 0x4000000;
```

Comment:
The values here are only for Windows and z/TPF; on other platforms the values are different. Same goes for the repeated definitions in `NodeGPF.java` and `PlatformSectionParser.java`.

Ideally, we would use the values discovered during the build, but the legacy DDR tooling doesn't support the expressions involved.

Reply:
Does this mean an operating-system specific `IProcess.resolveGenericSignal` needs to be implemented?